src/View/View.php

```diff
@@ -675,11 +669,9 @@ protected function createFileName($type, $parts = array())
 	protected function find($paths, $file)
 	{
 		$paths = is_array($paths) ? $paths : array($paths);
-
 		foreach ($paths as $path)
 		{
-			$fullname = $path . DIRECTORY_SEPARATOR . $file;
-
+			$fullname = $path . $file;
```

Comment:
Looks like `DIRECTORY_SEPARATOR` was removed but I'm not seeing why. Maybe I'm just missing it?

Reply:
I intentionally removed it.  All paths that the "find" method utilizes already have a separator added, since all paths are added via addPath which adds the directory separator here if it is needed.